GBA/gba.zig

```diff
@@ -13,6 +13,7 @@ pub const interrupt = @import("interrupt.zig");
 pub const math = @import("math.zig");
 pub const mem = @import("mem.zig");
 pub const obj = @import("obj.zig");
+pub const timer = @import("timer.zig");
```

Comment:
```suggestion
pub const timer = @import("timer.zig").timer;
```

Just made necessary by switching to an array.